app/lib/mail_publish_audit_event_observer.rb

```diff
@@ -0,0 +1,24 @@
+class MailPublishAuditEventObserver
+  def self.delivered_email(message)
+    unless message.respond_to?(:govuk_notify_personalisation)
+      Rails.logger.error "event=email_not_auditable_error|#{message.inspect}"
+      return
+    end
+
+    params = message.govuk_notify_personalisation
+    Rails.logger.error "event=audit_event_created,nomis_offender_id=#{params[:nomis_offender_id]}|#{message.inspect}"
```

Comment:
Minor, but should this be `Rails.logger.info`?

Reply:
Unfortunately "info" doesn't go into the logs, only error does. Rather than reconfigure the platform setup I just did this :joy: 😢